types/qunit/index.d.ts

```diff
@@ -693,3 +693,7 @@ interface QUnit {
 
 /* QUnit */
 declare const QUnit: QUnit;
+
+declare module 'qunit' {
+    export = QUnit;
```

Comment:
You can't reassign the export because everything declared here is already auto-exported. Try this instead:
```suggestion
    export default interface QUnitDefault extends QUnit {}
```

Reply:
@jamescdavis - I think have to use `export =` because the module actually **does not** have a default export.

Reply:
@jamescdavis everything outside this `declare module` block is globally-scoped (because there's no `export`)

Reply:
@dfreeman, yeah, I was wrong about why `export = QUnit` was erroring: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/47702#issuecomment-694403193

> You can invert that by wrapping the whole existing module in declare global and then moving this out into the top-level scope

I did exactly that in #47731 (as well as in `ember-qunit`).